env.d/60-flatpak

```diff
@@ -0,0 +1,15 @@
+# flatpak-environment-generator
+
+#!/bin/bash
+
+if [ "${XDG_DATA_DIRS#*flatpak}" = "${XDG_DATA_DIRS}" ]; then
+    OLD_XDG_DATA_DIRS=${XDG_DATA_DIRS}
+    XDG_DATA_DIRS="${XDG_DATA_HOME:-"$HOME/.local/share"}/flatpak/exports/share"
+    while read install_path
```

Comment:
This should be `while read -r install_path` to avoid misbehaving if a path with a backslash is printed.